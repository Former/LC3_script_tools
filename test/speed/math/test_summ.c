
int main()
{
    int i;
    int k;
    int res;
    int res1;

    for (res = -256; res <= 256; res++)
    {
        res1 = res;
        for (i = -128; i <= 128; i++)
        {
            for (k = -res1; k <= res1; k++)
                res += k;
        }
        printf("summ(%d, %d) = %d\n", res1, i, res);
    }

    return 0;
}
