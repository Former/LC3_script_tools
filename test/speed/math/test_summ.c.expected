summ(-256, 129) = -256
summ(-255, 129) = -255
summ(-254, 129) = -254
summ(-253, 129) = -253
summ(-252, 129) = -252
summ(-251, 129) = -251
summ(-250, 129) = -250
summ(-249, 129) = -249
summ(-248, 129) = -248
summ(-247, 129) = -247
summ(-246, 129) = -246
summ(-245, 129) = -245
summ(-244, 129) = -244
summ(-243, 129) = -243
summ(-242, 129) = -242
summ(-241, 129) = -241
summ(-240, 129) = -240
summ(-239, 129) = -239
summ(-238, 129) = -238
summ(-237, 129) = -237
summ(-236, 129) = -236
summ(-235, 129) = -235
summ(-234, 129) = -234
summ(-233, 129) = -233
summ(-232, 129) = -232
summ(-231, 129) = -231
summ(-230, 129) = -230
summ(-229, 129) = -229
summ(-228, 129) = -228
summ(-227, 129) = -227
summ(-226, 129) = -226
summ(-225, 129) = -225
summ(-224, 129) = -224
summ(-223, 129) = -223
summ(-222, 129) = -222
summ(-221, 129) = -221
summ(-220, 129) = -220
summ(-219, 129) = -219
summ(-218, 129) = -218
summ(-217, 129) = -217
summ(-216, 129) = -216
summ(-215, 129) = -215
summ(-214, 129) = -214
summ(-213, 129) = -213
summ(-212, 129) = -212
summ(-211, 129) = -211
summ(-210, 129) = -210
summ(-209, 129) = -209
summ(-208, 129) = -208
summ(-207, 129) = -207
summ(-206, 129) = -206
summ(-205, 129) = -205
summ(-204, 129) = -204
summ(-203, 129) = -203
summ(-202, 129) = -202
summ(-201, 129) = -201
summ(-200, 129) = -200
summ(-199, 129) = -199
summ(-198, 129) = -198
summ(-197, 129) = -197
summ(-196, 129) = -196
summ(-195, 129) = -195
summ(-194, 129) = -194
summ(-193, 129) = -193
summ(-192, 129) = -192
summ(-191, 129) = -191
summ(-190, 129) = -190
summ(-189, 129) = -189
summ(-188, 129) = -188
summ(-187, 129) = -187
summ(-186, 129) = -186
summ(-185, 129) = -185
summ(-184, 129) = -184
summ(-183, 129) = -183
summ(-182, 129) = -182
summ(-181, 129) = -181
summ(-180, 129) = -180
summ(-179, 129) = -179
summ(-178, 129) = -178
summ(-177, 129) = -177
summ(-176, 129) = -176
summ(-175, 129) = -175
summ(-174, 129) = -174
summ(-173, 129) = -173
summ(-172, 129) = -172
summ(-171, 129) = -171
summ(-170, 129) = -170
summ(-169, 129) = -169
summ(-168, 129) = -168
summ(-167, 129) = -167
summ(-166, 129) = -166
summ(-165, 129) = -165
summ(-164, 129) = -164
summ(-163, 129) = -163
summ(-162, 129) = -162
summ(-161, 129) = -161
summ(-160, 129) = -160
summ(-159, 129) = -159
summ(-158, 129) = -158
summ(-157, 129) = -157
summ(-156, 129) = -156
summ(-155, 129) = -155
summ(-154, 129) = -154
summ(-153, 129) = -153
summ(-152, 129) = -152
summ(-151, 129) = -151
summ(-150, 129) = -150
summ(-149, 129) = -149
summ(-148, 129) = -148
summ(-147, 129) = -147
summ(-146, 129) = -146
summ(-145, 129) = -145
summ(-144, 129) = -144
summ(-143, 129) = -143
summ(-142, 129) = -142
summ(-141, 129) = -141
summ(-140, 129) = -140
summ(-139, 129) = -139
summ(-138, 129) = -138
summ(-137, 129) = -137
summ(-136, 129) = -136
summ(-135, 129) = -135
summ(-134, 129) = -134
summ(-133, 129) = -133
summ(-132, 129) = -132
summ(-131, 129) = -131
summ(-130, 129) = -130
summ(-129, 129) = -129
summ(-128, 129) = -128
summ(-127, 129) = -127
summ(-126, 129) = -126
summ(-125, 129) = -125
summ(-124, 129) = -124
summ(-123, 129) = -123
summ(-122, 129) = -122
summ(-121, 129) = -121
summ(-120, 129) = -120
summ(-119, 129) = -119
summ(-118, 129) = -118
summ(-117, 129) = -117
summ(-116, 129) = -116
summ(-115, 129) = -115
summ(-114, 129) = -114
summ(-113, 129) = -113
summ(-112, 129) = -112
summ(-111, 129) = -111
summ(-110, 129) = -110
summ(-109, 129) = -109
summ(-108, 129) = -108
summ(-107, 129) = -107
summ(-106, 129) = -106
summ(-105, 129) = -105
summ(-104, 129) = -104
summ(-103, 129) = -103
summ(-102, 129) = -102
summ(-101, 129) = -101
summ(-100, 129) = -100
summ(-99, 129) = -99
summ(-98, 129) = -98
summ(-97, 129) = -97
summ(-96, 129) = -96
summ(-95, 129) = -95
summ(-94, 129) = -94
summ(-93, 129) = -93
summ(-92, 129) = -92
summ(-91, 129) = -91
summ(-90, 129) = -90
summ(-89, 129) = -89
summ(-88, 129) = -88
summ(-87, 129) = -87
summ(-86, 129) = -86
summ(-85, 129) = -85
summ(-84, 129) = -84
summ(-83, 129) = -83
summ(-82, 129) = -82
summ(-81, 129) = -81
summ(-80, 129) = -80
summ(-79, 129) = -79
summ(-78, 129) = -78
summ(-77, 129) = -77
summ(-76, 129) = -76
summ(-75, 129) = -75
summ(-74, 129) = -74
summ(-73, 129) = -73
summ(-72, 129) = -72
summ(-71, 129) = -71
summ(-70, 129) = -70
summ(-69, 129) = -69
summ(-68, 129) = -68
summ(-67, 129) = -67
summ(-66, 129) = -66
summ(-65, 129) = -65
summ(-64, 129) = -64
summ(-63, 129) = -63
summ(-62, 129) = -62
summ(-61, 129) = -61
summ(-60, 129) = -60
summ(-59, 129) = -59
summ(-58, 129) = -58
summ(-57, 129) = -57
summ(-56, 129) = -56
summ(-55, 129) = -55
summ(-54, 129) = -54
summ(-53, 129) = -53
summ(-52, 129) = -52
summ(-51, 129) = -51
summ(-50, 129) = -50
summ(-49, 129) = -49
summ(-48, 129) = -48
summ(-47, 129) = -47
summ(-46, 129) = -46
summ(-45, 129) = -45
summ(-44, 129) = -44
summ(-43, 129) = -43
summ(-42, 129) = -42
summ(-41, 129) = -41
summ(-40, 129) = -40
summ(-39, 129) = -39
summ(-38, 129) = -38
summ(-37, 129) = -37
summ(-36, 129) = -36
summ(-35, 129) = -35
summ(-34, 129) = -34
summ(-33, 129) = -33
summ(-32, 129) = -32
summ(-31, 129) = -31
summ(-30, 129) = -30
summ(-29, 129) = -29
summ(-28, 129) = -28
summ(-27, 129) = -27
summ(-26, 129) = -26
summ(-25, 129) = -25
summ(-24, 129) = -24
summ(-23, 129) = -23
summ(-22, 129) = -22
summ(-21, 129) = -21
summ(-20, 129) = -20
summ(-19, 129) = -19
summ(-18, 129) = -18
summ(-17, 129) = -17
summ(-16, 129) = -16
summ(-15, 129) = -15
summ(-14, 129) = -14
summ(-13, 129) = -13
summ(-12, 129) = -12
summ(-11, 129) = -11
summ(-10, 129) = -10
summ(-9, 129) = -9
summ(-8, 129) = -8
summ(-7, 129) = -7
summ(-6, 129) = -6
summ(-5, 129) = -5
summ(-4, 129) = -4
summ(-3, 129) = -3
summ(-2, 129) = -2
summ(-1, 129) = -1
summ(0, 129) = 0
summ(1, 129) = 1
summ(2, 129) = 2
summ(3, 129) = 3
summ(4, 129) = 4
summ(5, 129) = 5
summ(6, 129) = 6
summ(7, 129) = 7
summ(8, 129) = 8
summ(9, 129) = 9
summ(10, 129) = 10
summ(11, 129) = 11
summ(12, 129) = 12
summ(13, 129) = 13
summ(14, 129) = 14
summ(15, 129) = 15
summ(16, 129) = 16
summ(17, 129) = 17
summ(18, 129) = 18
summ(19, 129) = 19
summ(20, 129) = 20
summ(21, 129) = 21
summ(22, 129) = 22
summ(23, 129) = 23
summ(24, 129) = 24
summ(25, 129) = 25
summ(26, 129) = 26
summ(27, 129) = 27
summ(28, 129) = 28
summ(29, 129) = 29
summ(30, 129) = 30
summ(31, 129) = 31
summ(32, 129) = 32
summ(33, 129) = 33
summ(34, 129) = 34
summ(35, 129) = 35
summ(36, 129) = 36
summ(37, 129) = 37
summ(38, 129) = 38
summ(39, 129) = 39
summ(40, 129) = 40
summ(41, 129) = 41
summ(42, 129) = 42
summ(43, 129) = 43
summ(44, 129) = 44
summ(45, 129) = 45
summ(46, 129) = 46
summ(47, 129) = 47
summ(48, 129) = 48
summ(49, 129) = 49
summ(50, 129) = 50
summ(51, 129) = 51
summ(52, 129) = 52
summ(53, 129) = 53
summ(54, 129) = 54
summ(55, 129) = 55
summ(56, 129) = 56
summ(57, 129) = 57
summ(58, 129) = 58
summ(59, 129) = 59
summ(60, 129) = 60
summ(61, 129) = 61
summ(62, 129) = 62
summ(63, 129) = 63
summ(64, 129) = 64
summ(65, 129) = 65
summ(66, 129) = 66
summ(67, 129) = 67
summ(68, 129) = 68
summ(69, 129) = 69
summ(70, 129) = 70
summ(71, 129) = 71
summ(72, 129) = 72
summ(73, 129) = 73
summ(74, 129) = 74
summ(75, 129) = 75
summ(76, 129) = 76
summ(77, 129) = 77
summ(78, 129) = 78
summ(79, 129) = 79
summ(80, 129) = 80
summ(81, 129) = 81
summ(82, 129) = 82
summ(83, 129) = 83
summ(84, 129) = 84
summ(85, 129) = 85
summ(86, 129) = 86
summ(87, 129) = 87
summ(88, 129) = 88
summ(89, 129) = 89
summ(90, 129) = 90
summ(91, 129) = 91
summ(92, 129) = 92
summ(93, 129) = 93
summ(94, 129) = 94
summ(95, 129) = 95
summ(96, 129) = 96
summ(97, 129) = 97
summ(98, 129) = 98
summ(99, 129) = 99
summ(100, 129) = 100
summ(101, 129) = 101
summ(102, 129) = 102
summ(103, 129) = 103
summ(104, 129) = 104
summ(105, 129) = 105
summ(106, 129) = 106
summ(107, 129) = 107
summ(108, 129) = 108
summ(109, 129) = 109
summ(110, 129) = 110
summ(111, 129) = 111
summ(112, 129) = 112
summ(113, 129) = 113
summ(114, 129) = 114
summ(115, 129) = 115
summ(116, 129) = 116
summ(117, 129) = 117
summ(118, 129) = 118
summ(119, 129) = 119
summ(120, 129) = 120
summ(121, 129) = 121
summ(122, 129) = 122
summ(123, 129) = 123
summ(124, 129) = 124
summ(125, 129) = 125
summ(126, 129) = 126
summ(127, 129) = 127
summ(128, 129) = 128
summ(129, 129) = 129
summ(130, 129) = 130
summ(131, 129) = 131
summ(132, 129) = 132
summ(133, 129) = 133
summ(134, 129) = 134
summ(135, 129) = 135
summ(136, 129) = 136
summ(137, 129) = 137
summ(138, 129) = 138
summ(139, 129) = 139
summ(140, 129) = 140
summ(141, 129) = 141
summ(142, 129) = 142
summ(143, 129) = 143
summ(144, 129) = 144
summ(145, 129) = 145
summ(146, 129) = 146
summ(147, 129) = 147
summ(148, 129) = 148
summ(149, 129) = 149
summ(150, 129) = 150
summ(151, 129) = 151
summ(152, 129) = 152
summ(153, 129) = 153
summ(154, 129) = 154
summ(155, 129) = 155
summ(156, 129) = 156
summ(157, 129) = 157
summ(158, 129) = 158
summ(159, 129) = 159
summ(160, 129) = 160
summ(161, 129) = 161
summ(162, 129) = 162
summ(163, 129) = 163
summ(164, 129) = 164
summ(165, 129) = 165
summ(166, 129) = 166
summ(167, 129) = 167
summ(168, 129) = 168
summ(169, 129) = 169
summ(170, 129) = 170
summ(171, 129) = 171
summ(172, 129) = 172
summ(173, 129) = 173
summ(174, 129) = 174
summ(175, 129) = 175
summ(176, 129) = 176
summ(177, 129) = 177
summ(178, 129) = 178
summ(179, 129) = 179
summ(180, 129) = 180
summ(181, 129) = 181
summ(182, 129) = 182
summ(183, 129) = 183
summ(184, 129) = 184
summ(185, 129) = 185
summ(186, 129) = 186
summ(187, 129) = 187
summ(188, 129) = 188
summ(189, 129) = 189
summ(190, 129) = 190
summ(191, 129) = 191
summ(192, 129) = 192
summ(193, 129) = 193
summ(194, 129) = 194
summ(195, 129) = 195
summ(196, 129) = 196
summ(197, 129) = 197
summ(198, 129) = 198
summ(199, 129) = 199
summ(200, 129) = 200
summ(201, 129) = 201
summ(202, 129) = 202
summ(203, 129) = 203
summ(204, 129) = 204
summ(205, 129) = 205
summ(206, 129) = 206
summ(207, 129) = 207
summ(208, 129) = 208
summ(209, 129) = 209
summ(210, 129) = 210
summ(211, 129) = 211
summ(212, 129) = 212
summ(213, 129) = 213
summ(214, 129) = 214
summ(215, 129) = 215
summ(216, 129) = 216
summ(217, 129) = 217
summ(218, 129) = 218
summ(219, 129) = 219
summ(220, 129) = 220
summ(221, 129) = 221
summ(222, 129) = 222
summ(223, 129) = 223
summ(224, 129) = 224
summ(225, 129) = 225
summ(226, 129) = 226
summ(227, 129) = 227
summ(228, 129) = 228
summ(229, 129) = 229
summ(230, 129) = 230
summ(231, 129) = 231
summ(232, 129) = 232
summ(233, 129) = 233
summ(234, 129) = 234
summ(235, 129) = 235
summ(236, 129) = 236
summ(237, 129) = 237
summ(238, 129) = 238
summ(239, 129) = 239
summ(240, 129) = 240
summ(241, 129) = 241
summ(242, 129) = 242
summ(243, 129) = 243
summ(244, 129) = 244
summ(245, 129) = 245
summ(246, 129) = 246
summ(247, 129) = 247
summ(248, 129) = 248
summ(249, 129) = 249
summ(250, 129) = 250
summ(251, 129) = 251
summ(252, 129) = 252
summ(253, 129) = 253
summ(254, 129) = 254
summ(255, 129) = 255
summ(256, 129) = 256
